lib/containers/common.pm

```diff
@@ -265,7 +265,7 @@ sub test_rpm_db_backend {
     my ($running_version, $sp, $host_distri) = get_os_release("$runtime run $image");
     record_info('RPM check', "The rpm db backend in the image $image is $backend");
     # TW and SLE 15-SP3+ uses rpm-ndb in the image
-    if ($host_distri eq 'opensuse-tumbleweed' || ($host_distri eq 'sles' && check_version('>=15-SP3', "$running_version-SP$sp", qr/\d{2}(?:-sp\d)?/))) {
+    if ($host_distri eq 'opensuse-tumbleweed' || ($host_distri =~ /(sles|centos)/ && check_version('>=15-SP3', "$running_version-SP$sp", qr/\d{2}(?:-sp\d)?/))) {
         $expected = 'ndb';
     }
     die("The rpm db backend should be $expected") if ($backend ne $expected);
```

Comment:
maybe we even need to change this condition as well... cause centos is throwing some other extra character
https://openqa.suse.de/tests/5955362#step/podman_image/1

Reply:
```suggestion
    die("The rpm db backend should be $expected") if ($expected !~ m/$backend/);
```

can you try this?